### ISSUE TITLE
Address "Click" option match number

### ISSUE THREAD

**Issue by elescamilla** [lanl/Memento-Tracer-Extention#16]
When using the "click" option, the user may be presented with "CSSSelector, 9 matches". However, the trace will only capture one of the matches. That is not reflected to the user when creating the trace in the UI. 

For this issue:
* Which of the 9 matches is captured by the Trace? The first one or the one that was selected?
* Modify what is conveyed to the user to clarify that only 1 item will be Traced

**Comment by luda171** [comment]
It is not true. If the user selects click once and sees that CSSSelector has 9 matches, he should select XPATH.

**Comment by luda171** [comment]
I did put a workaround for "click until"  to click on the first element of CSS selector. We can highlight red the first element in this case.

**Comment by martinklein0815** [comment]
@elescamilla maybe test and see what needs to be done here?

**Comment by talyacooper** [comment]
I think the difference between "click all links in an area" and "click element" (which CAN result in multiple elements being clicked by the crawler) is somewhat confusing and hard to document. Here's an example:

For example, here I've captured 3 attempts at capturing all the links on one page of issues:
Attempt 1: I do "click once," click on the title of an issue and then select the css option that lists 12 matches. This willcrawl 12 matches

![Screen Shot 2022-09-07 at 12 24 29 PM](https://user-images.githubusercontent.com/54854192/189753248-da1eecac-f80d-4ea6-8d25-07cf9c4a1624.png)

Attempt 2:  I do "click all links in an area" and click on the title of an issue (the same place where I clicked above). It doesn't list any matches, so there is nothing to crawl.
![Screen Shot 2022-09-07 at 12 28 06 PM](https://user-images.githubusercontent.com/54854192/189753448-beb68fbd-5ed3-4968-af43-437be7ff6b5c.png)

Attempt 3: I do "click all links in an area" and click in the wider box around the issue title. That will select 12 issues links, as well as all the tags and authors around them. 
![Screen Shot 2022-09-07 at 12 27 14 PM](https://user-images.githubusercontent.com/54854192/189754024-3e5039db-8a0a-41ce-a651-c77b154c35fa.png)

Basically, either "click" or "click all links in an area" can be used to select a group of links in a given area. How do we define the difference and guide a user to which is more useful?

FWIW Luda's changes to "click all in area" may have made my initial observation to Emily no longer relevant, but I still feel puzzled over the larger click vs click all functionality.

**Comment by luda171** [comment]
I would say  “click all links in area “  is special simplified not generic
macro - we can call it “download all links in area”? It collects urls  from
href attributes.
It is also not disrupting navigation, since download happens later. The
“click” action is more generic , and assumes that crawler actually perform
click action on the go. It can change state of the page. The elements to
click can be anything “clickable” including link. So user need to think
more what will happen if he select multiple elements to click. It is also
can have sub-events or children events. Yes, in some cases you can produce
same results using different action items.

**Comment by talyacooper** [comment]
This is still very confusing but I think will be best addressed with documentation for the time being.
